framework/ui/view/theme.cpp

```diff
@@ -243,3 +247,22 @@ void Theme::initMusicalFont()
         update();
     });
 }
+
+void Theme::setupWidgetTheme()
```

Comment:
I think this function needs to be in some other class. Theme must simply teel to us which of color/fonts used in the current moment. It must not configuring all system

Reply:
decided to leave it as it is